### DISCUSSION TITLE
希望能够支持按上传时间筛选图片

### DISCUSSION THREAD

**Discussion by orilights** [lsky-org/lsky-pro#424] [想法]
首先感谢一下作者开发这款程序。

目前我是将兰空图床当做博客的图片管理来使用的，因为Typecho的附件管理过于简陋。

在图片较多时，寻找比较久远的图片会变得较为困难，所以希望作者能够考虑一下支持按上传时间（年-月）来筛选图片的方式。

**Comment by pdkst** [comment]
同求，我希望画廊和相册有时间线的功能，按照时间排序，类似这种
![image](https://user-images.githubusercontent.com/8915631/170864366-5718c4e7-20b9-4f56-ad0e-15a27e3406bc.png)
